.circleci/config.yml

```diff
@@ -308,6 +314,40 @@ commands:
             done;
             exit 0
 
+  update-dispatch-rules:
+    parameters:
+      config_repo_path:
+        type: string
+        default: *github_dispatch_repo_path
+      config_file_path:
+        type: string
+        default: *github_dispatch_config_path
+      config_branch:
+        type: string
+        default: *github_dispatch_config_branch
+    steps:
+      - run:
+          name: Get dispatch.yaml file
+          command: |
+            # We are going to copy the dispatch file from the other repo
+            set -u
+            API_BASE_URL='https://api.github.com/repos'
+            API_BASE_PATH="<< parameters.config_repo_path >>/contents"
+            FILE_PATH="<< parameters.config_file_path >>/${ENVIRONMENT}/dispatch.yaml"
+            FILE_URL="${API_BASE_URL}${API_BASE_PATH}${FILE_PATH}"
+            BRANCH="<< parameters.config_branch >>"
+            FULL_URL="${FILE_URL}?ref=${BRANCH}"
+            curl /
+              -H 'Accept:application/vnd.github.v3.raw' /
+              "$FULL_URL" /
+              -o dispatch.yaml
+            cat dispatch.yaml
```

Comment:
Where we get the new dispatch.yaml file from the angular repo.

Reply:
Niiice